includes/Admin/Admin_Page.php

```diff
@@ -194,7 +194,8 @@ public function render_page() {
 
 		$selected_plugin_basename = filter_input( INPUT_GET, 'plugin', FILTER_SANITIZE_FULL_SPECIAL_CHARS );
 
-		$categories = Check_Categories::get_categories();
+		$categories      = Check_Categories::get_categories();
+		$category_labels = Check_Categories::get_category_labels();
```

Comment:
Nit-pick, but this could be simplified to only use `$category_labels`. It already includes the categories as the keys.

We could then also simplify the `admin-page.php` template, to iterate through `$category_labels` for access to both the slugs and the labels, instead of iterating through `$categories` and accessing the labels from the other variable.